write-api-client/src/main/java/json/Output.java

```diff
@@ -0,0 +1,46 @@
+package json;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * @author gluu
```

Comment:
Describe your classes, I'd guess that this is for retaining information between your two CLI commands, but this should be recorded